### PR TITLE
Add logic to create entity views from template

### LINKED ISSUE #50: Client support for entity view creation
There are a number of entity types supported by the SynapseCreation class which aren't supported when creating resources from a template. The one I need ASAP is entity view, so I will implement that.

### PR DESCRIPTION
**Pull request by philerooski**
Resolves #50

### CODE REVIEW

**Comment by thomasyu888** [comment]
Thanks Phil - can you add to this template: https://github.com/Sage-Bionetworks/synapseformation/blob/main/templates/long.yaml

**Comment by philerooski** [comment]
> Thanks Phil - can you add to this template: https://github.com/Sage-Bionetworks/synapseformation/blob/main/templates/long.yaml

@thomasyu888 done

File: templates/long.yaml [hunk 1]
```diff
@@ -38,7 +38,14 @@
   can_public_join: false
   description: Test team description
   invitations:
-  - message: Welcome to the Test Team! 
+  - message: Welcome to the Test Team!
     members:
     - principal_id: 3426116
     - email: synapseformation-test-user@sagebase.org
+
+- name: Test Entity View
```
Comment (thomasyu888) [inline templates/long.yaml]: This should be under a project as a children.

Comment (philerooski) [reply templates/long.yaml]: Okay, I amended my commit.

**Review by thomasyu888 — APPROVED** [review]

**Pull request merged**